mldsa/sign.c

```diff
@@ -96,6 +171,12 @@ int crypto_sign_keypair_internal(uint8_t *pk, uint8_t *sk,
   /* Compute H(rho, t1) and write secret key */
   shake256(tr, MLDSA_TRBYTES, pk, CRYPTO_PUBLICKEYBYTES);
   mld_pack_sk(sk, rho, tr, key, &t0, &s1, &s2);
+
+  /* Pairwise Consistency Test (PCT) @[FIPS140_3_IG, p.87] */
+  if (mld_check_pct(pk, sk))
+  {
+    return -1;
```

Comment:
You need to update the comments and contracts on crypto_sign_keypair_internal() and crypto_sign_keypair() to indicate that they can return 0 (success) or (now) -1 (indicating that the PCT has failed).